src/algebraic_geometry/properties.lean

```diff
@@ -83,4 +83,49 @@ begin
     { intro x, exact x.rec _ } }
 end
 
+lemma has_generic_point [is_integral X] :
+  ∃ (x : X.carrier), closure ({x} : set X.carrier) = ⊤ :=
```

Comment:
Should we have a definition `generic_point x s` in the topology library, with a bit of API around it (e.g. specialisation of points, etc.)?

Reply:
We should certainly develop the notion of specialization for more general topological spaces.

Reply:
Indeed. I think #10914 should be enough for now.

Reply:
Great! Sorry I'm just now catching up on some of these PRs.